plugin/pkg/admission/alwayspullimages/admission_test.go

```diff
@@ -93,6 +93,31 @@ func TestValidate(t *testing.T) {
 	}
 }
 
+func TestValidateAdmittedInitContainers(t *testing.T) {
+	namespace := "test"
+	handler := &AlwaysPullImages{}
+	pod := api.Pod{
```

Comment:
@fntlnz thx for contributing this. 

i suppose we can fold this test into `TestValidate`^^^ as one test case. wdyt?

Reply:
yes makes sense, thanks for the suggestion